Dependencies/napi/napi-jsi/include/napi/napi.h

```diff
@@ -1161,6 +1130,8 @@ namespace Napi {
     CallbackInfo(napi_env env, const jsi::Value& thisVal, const jsi::Value* args, size_t argc, const jsi::Value& newTarget, void* data);
     ~CallbackInfo();
 
+    CallbackInfo(CallbackInfo &&) = default;
```

Comment:
This is going to cause `_dynamicArgs` to be deleted twice I think. If we must have a move constructor, it needs to be implemented properly. Also, it's weird to have just a move constructor and no move assignment.